lib/cadet/accounts/ivle.ex

```diff
@@ -82,15 +85,14 @@ defmodule Cadet.Accounts.IVLE do
 
   """
   def fetch_role(token) do
-    {:ok, modules} = api_fetch("Modules", AuthToken: token, CourseID: "CS1101S")
+    {:ok, modules} = api_call("Modules", AuthToken: token, CourseID: "CS1101S")
 
     cs1101s =
```

Comment:
Not sure if the 
`cs1101s = modules["Results"] |> Enum.find` part should be refactored to a separate method. Currently there are only 2 occurrences of them, so let's keep them separate for now following the rule of 3